infrastructure-formula/infrastructure/salt/minion_networkautomation.sls

```diff
@@ -0,0 +1,43 @@
+{%- set mypillar = salt['pillar.get']('infrastructure:salt:odd_proxy', {}) -%}
+
+{%- if mypillar %}
+include:
+  - .proxy_networkautomation
+
+salt_odd_proxy_config:
+  file.managed:
+    - names:
+      - /etc/salt-pod/minion:
+        - contents: |
+            # Managed by Salt
+            {%- if 'master' in mypillar %}
+            master:
+              {%- for master in mypillar['master'] %}
+              - {{ master }}
+              {%- endfor %}
+            {%- endif %}
+            log_level: info
+            saltenv: production_network
```

Comment:
That looks like a hardcoded git branch name. What about making it a pillar (maybe using `mypillar.get('foo', 'production_network'` so that there is a good default)?

Reply:
Saltenvs do not necessarily need to be mapped to Git branches. In this case it points to a local directory on the Salt master. The configuration pillar will make it make sense.

Reply:
Will leave it for now as we need it for a project, but I'm happy to revisit it if you have more input later on.